src/mrpro/data/_KData.py

```diff
@@ -149,7 +148,7 @@ def reshape_acq_data(data):
             case KTrajectoryCalculator():
                 ktraj_calc = ktrajectory(kheader)
                 if isinstance(ktraj_calc, KTrajectoryRawShape):
-                    ktraj = ktraj_calc.reshape(sort_idx, num_k2, num_k1)
+                    ktraj = ktraj_calc.reshape(sort_idx, num_k2, num_k1, repeat_detection_tolerance=None)
```

Comment:
Why is `repeat_detection_tolerance=None` needed? @schuenke I think this caused the problems you had when trying out Andreas' `FourierOp`

Reply:
in _KTrajectory there is a check for this parameter. We had to set it to None. Default was a float. I honestly cannot tell you anymore why exactly it has to be done..